erpnext/controllers/status_updater.py

```diff
@@ -118,6 +118,7 @@ class StatusUpdater(Document):
 	def update_prevdoc_status(self):
 		self.update_qty()
 		self.validate_qty()
+		self.validate_rate()
```

Comment:
Is this related to this fix?

Reply:
Yeah, it's for the first issue I'd mentioned in the PR description. Vrinda Oza had pointed out that the amount was positive in her return invoice, even though it was supposed to be negative. It was because she'd entered the rate(and not just the quantity) as negative.